### DISCUSSION TITLE
How do I update CharacterVirtual?

### DISCUSSION THREAD

**Discussion by OenaZ** [amerkoleci/JoltPhysicsSharp#107] [Q&A] [answered]
I'm new to using Jolt so I am still trying to wrap my head around it but I keep getting "System.AccessViolationException" when trying to use CharacterVirtual's Update Method. So far that error has only appeared when I am missing something, how do I fix this?

**Comment by amerkoleci** [comment]
Can you please share repro code?

Thanks

**Comment by OenaZ** [comment, marked as answer]
Hello, I apologize for the late response but it turns out that the problem was simply because I didn't initialize "Shape" within the "CharacterVirtualSettings".
